source/examples/xgboost-randomforest-gpu-hpo-dask/notebook.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>The url for train_test_split is really old. Can you update to:https://docs.rapids.ai/api/cuml/stable/api.html#cuml.model_selection.train_test_split</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>